docs/get-started-with-datahub.md

```diff
@@ -0,0 +1,62 @@
+# Get Started With DataHub
+
+This section is focused on helping you get up and running with DataHub as quickly as possible.
+
+## Get Started with Self-Hosted DataHub
+
+This is a great place to start if you're new to DataHub and are interested in taking the open-source project for a spin.
+
+You'll learn how to:
+
+* Deploy DataHub on your local machine
+* Ingest sample data to get a feel for what is possible in DataHub
+* Add and invite users to DataHub
+* Troubleshoot common issues
+
+### Prerequisites
+
+Before you go further, ensure you have the following installed:
+
+* [Docker](https://docs.docker.com/get-docker/)
+* [jq](https://stedolan.github.io/jq/download/)
+* [Docker Compose](https://github.com/docker/compose/blob/master/INSTALL.md) - if using Linux
+
+:::note
+
+Make sure to allocate enough hardware resources for Docker! 
+We recommend the following configuration: 2 CPUs, 8GB RAM, 2GB Swap area, and 10GB disk space.
+
+:::
+
+<p>
+<a
+    className='button button--primary button--lg'
+    href="quickstart">
+    Get Started with Self-Hosted DataHub
+</a>
+</p>
+
+## Get Started with Managed DataHub
+
+[Acryl Data](https://www.acryldata.io/product) provides a fully managed, premium version of DataHub.
+
+In this section, you'll learn how to:
+
+* Add and invite users to DataHub
+* Configure Slack notifications
+* Understand Approval Workflows
```

Comment:
Same as above -- We'll need to update this list as the scope of the section expands

Reply:
Should we add set up SSO here?

Reply:
As an alternative to add and invite users

Reply:
@aditya-radhakrishnan we haven't migrated SaaS-specific SSO configuration docs yet, so I'm a little hesitant to add it in until we do

Reply:
Makes sense!